source/core/parameters_lagrangian.cc

```diff
@@ -598,6 +598,70 @@ namespace Parameters
       prm.leave_subsection();
     }
 
+    template <int dim>
+    void
+    ForceTorqueOnWall<dim>::declare_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection("Forces and Torques");
+      prm.declare_entry("calculation",
+                        "false",
+                        Patterns::Bool(),
+                        "Enable calculation of forces");
+      prm.declare_entry(
+        "verbosity",
+        "verbose",
+        Patterns::Selection("quiet|verbose"),
+        "State whether output from solver runs should be printed. "
+        "Choices are <quiet|verbose>.");
+      prm.declare_entry("Output name of force and torque's file (string)",
+                        "force",
+                        Patterns::FileName(),
+                        "File output force prefix");
+      prm.declare_entry("Which output's creation frequency? (int)",
+                        "1",
+                        Patterns::Integer(),
+                        "Output frequency");
+      prm.declare_entry("Coordinate x of center of mass",
+                        "0",
+                        Patterns::Double(),
+                        "X coordinate of center of mass");
+      prm.declare_entry("Coordinate y of center of mass",
+                        "0",
+                        Patterns::Double(),
+                        "Y coordinate of center of mass");
+      prm.declare_entry("Coordinate z of center of mass",
+                        "0",
+                        Patterns::Double(),
+                        "Z coordinate of center of mass");
+      prm.leave_subsection();
+    }
+
+    template <int dim>
+    void
+    ForceTorqueOnWall<dim>::parse_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection("Forces and Torques");
+      calculate_force_torque    = prm.get_bool("calculation");
+      const std::string verbose = prm.get("verbosity");
+      if (verbose == "quiet")
+        force_torque_verbosity = Verbosity::quiet;
+      else if (verbose == "verbose")
+        force_torque_verbosity = Verbosity::verbose;
+      else
+        {
+          throw(std::runtime_error("Invalid verbosity choice "));
+        }
+      force_torque_output_name =
+        prm.get("Output name of force and torque's file (string)");
+      output_frequency =
+        prm.get_integer("Which output's creation frequency? (int)");
```

Comment:
I think it is better to reword the sentence within the quotation marks.

Reply:
Avoid question marks and etc in the text file also. You can look in the wiki for other examples

Reply:
force_torque_output_name =
        prm.get("Name file");
      output_frequency =
        prm.get_integer("Output frequency");
Better?